index.html

```diff
@@ -12,14 +12,24 @@
   <body>
     <header id="store">
       <h1>Greengrocers</h1>
+      <div class="filter-buttons">
+        <button class="fruit-filter">Show Fruits</button>
+        <button class="veg-filter">Show Vegetables</button>
+        <button class="clear-filter">Show All</button>
+      </div>  
```

Comment:
You have modified the HTML? I think that the HTML could be constructed only using JSDOM functions...

Reply:
Probably, but I wanted to position them in a specific place so it was far easier to simply modify the HTML and CSS slightly.